invision-app/app/components/addEditMovie/addEditMovie.js

```diff
@@ -1,74 +1,75 @@
-import React, { useState, useEffect } from 'react';
-import './addMovie.scss'
+import React from 'react';
+import './addMovie.scss';
+import { useFormik } from 'formik';
 import PropTypes from 'prop-types';
 
-export function AddEditMovie(props) {
-
-  const [newMovie, setMovie] = useState({
-    id: props?.movie?.id || Math.floor(Math.random() + Date.now()),
-    title: props?.movie?.title,
-    release_date: props?.movie?.release_date,
-    poster_path: props?.movie?.poster_path,
-    genres: props?.movie?.genres,
-    overview: props?.movie?.overview,
-    runtime: props?.movie?.runtime
-  });
+const validate = (values) => {
```

Comment:
Validation should showes errors first time only after user click on **submit** button. Right now it is very strange from user experience percpective when I just start filling form and avery field in this form are red. Also, it would be great to add error messages that can explain what is wrong to the fields.